build/builder.sh

```diff
@@ -79,8 +87,8 @@ cockroach_toplevel="$(dirname $(cd $(dirname $0); pwd))"
 docker run -i ${tty-} ${rm} \
   --volume="${gopath0}/src:/go/src" \
   --volume="${gopath0}/pkg:/go/pkg" \
-  --volume="${gopath0}/pkg/linux_amd64_race:/usr/src/go/pkg/linux_amd64_race" \
-  --volume="${gopath0}/bin/linux_amd64:/go/bin" \
+  --volume="${gopath0}/pkg/${goos}_amd64_race:/usr/src/go/pkg/${goos}_amd64_race" \
```

Comment:
so this will be mounted inside the container at `/usr/src/go/pkg/linux_docker_amd64_race`? How will that be used?

Why not simply `--volume="${gopath0}/pkg/docker_amd64_race:/usr/src/go/pkg/linux_amd64_race"` here?